src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_provider.py

```diff
@@ -0,0 +1,20 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest
```

Comment:
No need to add this test in a separate file as you can use existing test file `test_resource.py`.